scripts/vagrant/setup.sh

```diff
@@ -40,7 +40,7 @@ echo "Ensuring Unix user '$SOLR_USER' exists"
 useradd $SOLR_USER || :
 DOWNLOAD_DIR='/dataverse/downloads'
 GLASSFISH_ZIP="$DOWNLOAD_DIR/glassfish-4.1.zip"
-SOLR_TGZ="$DOWNLOAD_DIR/solr-7.3.0.tgz"
+SOLR_TGZ="$DOWNLOAD_DIR/solr-7.3.1.tgz"
```

Comment:
For this to work we need to edit downloads/download.sh

Reply:
Another good catch.